bot.py

```diff
@@ -22,6 +22,13 @@
 RE_TAG_PROG = re.compile('([A-Za-z0-9_.-]+)?#(\d+)')
 RE_URL_PROG = re.compile('github.com/([A-Za-z0-9_.-]+)/([A-Za-z0-9_.-]+)/(issues|pulls)/(\d+)\S*')
 
+REPOSITORY_SHORT_NAMES = {
+    "assetlib": "godot-asset-library",
+    "demo": "godot-demo-projects",
+    "doc": "godot-docs",
```

Comment:
I would prefer `docs` instead of `doc`. It's probably what most people are going to try first.